packages/airnode-adapter/src/request-building/authentication.ts

```diff
@@ -104,8 +104,10 @@ function getSchemeAuthentication(
       return getRelayAuthSchemeFromMetadata(apiSecurityScheme, options, 'chainType');
     case 'relayRequesterAddress':
       return getRelayAuthSchemeFromMetadata(apiSecurityScheme, options, 'requesterAddress');
-    default:
-      return {};
```

Comment:
Is it ok to get rid of the `default`?

Reply:
Yeah, it should be verified by validator.
Also stripping away the default is cleaner, since TS will use checked switch statement and warn you when you whn you add a new auth scheme.